update docs to be more clear regarding audio support and go2rtc requi… (#16232)
blakeblackshear/frigate: docs/docs/troubleshooting/faqs.md

    record: preset-record-generic-audio-aac
```

> ### No audio in live stream or recordings in Web UI.
> Audio is only supported in the browser using webRTC or MSE APIs. Frigate only supports these APIs when go2rtc is configured so if you are 
> hoping to experience audio in the browser you need to enable and configure go2rtc.
> 

Reviewer: ```suggestion
### No audio in live stream or recordings in Web UI.

Audio is only supported for live view when go2rtc is configured, see [the live docs](../configuration/live.md) for more information.

```

    record: preset-record-generic-audio-aac
```

> ### No audio in live stream or recordings in Web UI.

Reviewer: ```suggestion
### No audio in live stream or recordings in Web UI.

Audio is only supported for live view when go2rtc is configured, see [the live docs](../configuration/live.md) for more information.```

Audio is only supported for live view when go2rtc is configured, see [the live docs](../configuration/live.md) for more information.```

> 
> 
> 

Reviewer: ```suggestion

```

    record: preset-record-generic-audio-aac
```

### No audio in live stream or recordings in Web UI.

Audio is only supported for live view when go2rtc is configured, see [the live docs](../configuration/live.md) for more information.```

> Audio is only supported in the browser using webRTC or MSE APIs. Frigate only supports these APIs when go2rtc is configured so if you are 
> hoping to experience audio in the browser you need to enable and configure go2rtc.

Reviewer: ```suggestion
```

    record: preset-record-generic-audio-aac
```

> ### No audio in live stream or recordings in Web UI.

Reviewer: ```suggestion
### How can I get sound in live view?
```

> Audio is only supported for live view when go2rtc is configured, see [the live docs](../configuration/live.md) for more information.```

Reviewer: ```suggestion
Audio is only supported for live view when go2rtc is configured, see [the live docs](../configuration/live.md) for more information.

```
